backend/Dockerfile

```
@@ -13,7 +13,7 @@ RUN aws configure set default.region ap-southeast-1
 WORKDIR /usr/home/postmangovsg
 
 COPY shared ../shared
-RUN cd ../shared && npm ci && cd ../postmangovsg
+RUN cd ../shared && npm ci
```

```suggestion
RUN cd ../shared && npm ci && npm run build
```

This should work too (haven't tested). Originally I didn't have to run `npm run build` in `shared` because the tsc command in `backend` will build all project references also. But in this case since there's post-build actions might be easier to explicitly run build.

---

Yep I was thinking about this, just that this will result in `shared` being built twice. I doubt it will have significant cost, but not sure if calling `npm run build` (and hence implicitly calling the postbuild script) is more readable or if running `postbuild` is more readable